### PR TITLE
Release 11.0.1

### PR DESCRIPTION
**Pull request by davidyuk**
supersede #1439

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ module.exports = {
       ignoreRegExpLiterals: true,
       ignoreStrings: true,
       ignoreTemplateLiterals: true
-    }]
+    }],
+    'import/no-cycle': 2
```
Comment (marc0olo) [inline .eslintrc.js:27]: what's that about? asking due to the linting issue in the pipeline

**Review by marc0olo — COMMENTED** [review]
@davidyuk what about the failing pipeline due to linting issues? should we ignore that?

otherwise I am fine.

**Comment by marc0olo** [comment]
@subhod-i @thepiwo can you also have a short look and provide your "ok"?

**Review by thepiwo — APPROVED** [review]
looks good as far as I can see

**Review by subhod-i — APPROVED** [review]
A minor linting issue. Rest everything is excellent.

**Pull request merged**